### ISSUE TITLE
Improve search across a collection

### ISSUE THREAD

**Issue by KimPatch** [DukeLivingHistory/RLH-Site-Code#37]
There is a search bar on the collection page that says search within collection, but it's only searching the headings and descriptions. What would it take to do a fulltext search across a collection?

**Comment by ethanbutler** [comment]
[This plugin](https://wordpress.org/plugins/acf-better-search/) should take care of it. As part of January maintenance, I can give it a a test run and install it on the site if it seems to do the job.

**Comment by KimPatch** [comment]
Multiword search looks good!

1. But I'm not seeing collection descriptions included in the primary search.

Test: Primary search, from
http://rlh.demo.sqnts.xyz/collections/
I searched on "legislation across", which are the second and third words of the Closing the Polls?… collection
I expected to get that first collection in the results but there are no results found

2. And I'm not seeing the improvements on the collection pages – maybe I'm misunderstanding this one.

Test: on a Collection page:
http://rlh.demo.sqnts.xyz/collections/north-carolina-voting-rights/
I searched on "as you know", which are the first words of the Jay Delancey interview, which appears in that collection and got no results
I searched on "community organizing", which is in the first Jay Delancey supporting content box, and got no results
I searched on "Supreme Court decision", which appears in the text of the page and got no results.
